graph/common/Encoding.java

```diff
@@ -257,25 +257,25 @@ public boolean isSystem() {
         PROPERTY_THEN(6),
         PROPERTY_VALUE(7),
         PROPERTY_VALUE_REF(8),
-        EDGE_ISA_IN(-40), // EDGE_ISA_OUT does not exist by design
-        EDGE_SUB_OUT(50),
-        EDGE_SUB_IN(-50),
-        EDGE_OWNS_OUT(51),
-        EDGE_OWNS_IN(-51),
-        EDGE_OWNS_KEY_OUT(52),
-        EDGE_OWNS_KEY_IN(-52),
-        EDGE_PLAYS_OUT(53),
-        EDGE_PLAYS_IN(-53),
-        EDGE_RELATES_OUT(54),
-        EDGE_RELATES_IN(-54),
-        EDGE_HAS_OUT(70),
-        EDGE_HAS_IN(-70),
-        EDGE_PLAYING_OUT(71),
-        EDGE_PLAYING_IN(-71),
-        EDGE_RELATING_OUT(72),
-        EDGE_RELATING_IN(-72),
-        EDGE_ROLEPLAYER_OUT(73, true),
-        EDGE_ROLEPLAYER_IN(-73, true);
+        EDGE_ISA_BACKWARD(-40), // EDGE_ISA_FORWARD does not exist by design
+        EDGE_SUB_FORWARD(50),
+        EDGE_SUB_BACKWARD(-50),
+        EDGE_OWNS_FORWARD(51),
+        EDGE_OWNS_BACKWARD(-51),
+        EDGE_OWNS_KEY_FORWARD(52),
+        EDGE_OWNS_KEY_BACKWARD(-52),
+        EDGE_PLAYS_FORWARD(53),
+        EDGE_PLAYS_BACKWARD(-53),
+        EDGE_RELATES_FORWARD(54),
+        EDGE_RELATES_BACKWARD(-54),
+        EDGE_HAS_FORWARD(70),
+        EDGE_HAS_BACKWARD(-70),
+        EDGE_PLAYING_FORWARD(71),
+        EDGE_PLAYING_BACKWARD(-71),
+        EDGE_RELATING_FORWARD(72),
+        EDGE_RELATING_BACKWARD(-72),
+        EDGE_ROLEPLAYER_FORWARD(73, true),
+        EDGE_ROLEPLAYER_BACKWARD(-73, true);
```

Comment:
Implement a TODO that indicated to rename `in` and `out` to `backward` and `forward`. These new names make the understanding of the `//graph` package much clearer!